### ISSUE TITLE
WiFi pass-through and NMEA broadcast

### ISSUE THREAD

**Issue by selimnairb** [CCOMJHC/WIBL#28]
**[Original report](https://bitbucket.org/ccomjhc/wibl/issues/26) by Brian Calder (Bitbucket: [brian_r_calder](https://bitbucket.org/brian_r_calder), ).**

----------------------------------------

If the WiFi radio is on, it should be possible to pass through all NMEA traffic onto a broadcast network \(ideally multicast to avoid traffic volume\) and therefore form a NMEA-WiFi bridge.

**Comment by brian-r-calder** [comment]
> [@brian-r-calder](https://github.com/brian-r-calder) Move to discussion?

Agreed: it would be good to determine whether this is something useful to do before making it a real issue.
